src/commands/start/common_option.py

```diff
@@ -202,7 +205,9 @@
         '--min-deposit-amount-gwei',
         type=int,
         envvar='MIN_DEPOSIT_AMOUNT_GWEI',
-        help='Minimum amount in gwei to deposit into validator.',
+        help=f'Minimum amount in gwei to deposit into validator.'
+        f' The default is {DEFAULT_MIN_DEPOSIT_AMOUNT_GWEI} '
+        f'({Web3.from_wei(Web3.to_wei(DEFAULT_MIN_DEPOSIT_AMOUNT_GWEI, 'gwei'), 'ether')} ETH).',
```

Comment:
Use `settings.DEFAULT_MIN_DEPOSIT_AMOUNT`
```suggestion
        f'({DEFAULT_MIN_DEPOSIT_AMOUNT} ETH).',
```